lib/src/configuration.dart

```diff
@@ -146,6 +149,8 @@ class Configuration {
     logLevel = configValues.logLevel; // or env. var. SENTRY_LOG_LEVEL
     binDir = configValues.binDir ?? '.dart_tool/pub/bin/sentry_dart_plugin';
     binPath = configValues.binPath;
+    sentryCliCdnUrl = configValues.sentryCliCdnUrl ??
+        'https://downloads.sentry-cdn.com/sentry-cli';
```

Comment:
When we provide the default path here, we'd never need the value from `_sources.dart`, as we always call the constructor where we pass this prefix.

We can probably just remove it from "_sources.dart" and just make `_formatDownloadUrl(String prefix)` public and remove `_formatDownloadUrl` getter.

Reply:
I have the same idea when I coding the PR.

Reply:
done.